### PR TITLE
Add Flux documentation

### PR DESCRIPTION
**Pull request by ivallhon**
*Description of changes:*

As per discussion with @kcaws , this PR adds documentation to bootstrap EKS clusters configuration with Flux using a sample repository. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/Advanced/gitops-with-flux.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Manage your cluster(s) configuration with Flux
+
+Once you have deployed your EKS cluster(s) with Terraform, you can leverage [Flux](https://fluxcd.io) to manage your cluster(s) configuration with [GitOps](https://www.gitops.tech/), including the deployment of add-ons, cluster configuration (e.g. cluster policies) and applications. Using GitOps practices to manage your clusters configuration will simplfy management, scaling the number of clusters you run and be able to easily recreate your clusters, treating them as ephemeral resources. Recreating your cluster is as simple as deploying a new cluster with Terraform and bootstrap it with Flux pointing to the repository containing the configuration.
```
Comment (kcoleman731) [inline docs/Advanced/gitops-with-flux.md:3]: to manage your cluster's configuration

Comment (kcoleman731) [reply docs/Advanced/gitops-with-flux.md:3]: with Terraform and bootstraping

File: docs/Advanced/gitops-with-flux.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+**NOTE:** The add-ons on the sample are not configured for a production-ready cluster (e.g. Prometheus would need to be configured for long term metric storage, nginx would need HPA and any custom settings you need...).
+
+There're also a set of Kyverno cluster policies deployed to audit (test) or enforce (production) security settings on your workloads, as well as [podinfo](https://github.com/stefanprodan/podinfo) as a sample application, configured with [Flagger](https://flagger.app/) to perform progressive deployments. For further information, visit the [aws-samples/ssp-flux-cluster-configuration](https://github.com/ivallhon/k8s-infra) repository documentation.
+
+## Bootstrap your cluster with Flux
+
+The below instructions assume you have created a cluster with aws-eks-accelerator-for-terraform with no add-ons other than aws-load-balancer-controller. If you're installing additonal add-ons via terraform, the configuration may clash with the one on the sample repository. As recommendation, use Terraform to install and manage only add-ons that require additional AWS resources to be created (like IAM roles for Service accounts), and then use Flux to manage the rest.
```
Comment (kcoleman731) [inline docs/Advanced/gitops-with-flux.md:24]: `aws-eks-accelerator-for-terraform`

Comment (kcoleman731) [inline docs/Advanced/gitops-with-flux.md:24]: If you plan to leverage Flux, we recommend that you use Terraform

**Review by kcoleman731 — COMMENTED** [review]
Looks great!

File: docs/Advanced/gitops-with-flux.md (hunk 2 above)
Comment (ivallhon) [reply docs/Advanced/gitops-with-flux.md]: done

Comment (ivallhon) [reply docs/Advanced/gitops-with-flux.md]: done

**Comment by ivallhon** [comment]
thanks for the review @kcoleman731 ! Updated docs per comments

**Review by kcoleman731 — APPROVED** [review]

**Pull request merged**